demos/common/cpp/models/include/models/detection_model_yolo.h

```diff
@@ -38,9 +38,17 @@ class ModelYolo : public DetectionModel {
         int outputHeight = 0;
 
         Region(const std::shared_ptr<ngraph::op::RegionYolo>& regionYolo);
+        Region(int classes, int coords, const std::vector<float>& anchors, const std::vector<int64_t>& masks, int outputWidth, int outputHeight);
     };
 
 public:
+    enum YoloVersion {
+        YOLO_V1V2,
+        YOLO_V3,
+        YOLO_V4,
+        YOLO_V4_TINY
+    };
```

Comment:
Since all of it is part of the `YOLO` class, I'd prefer:
```cpp
enum Version
{
    V2,
    V3,
    V4
}
```
Also, there is no need for separete tiny version.

Reply:
Please leave it as is, as V2 is to short for search